requirements.txt

```diff
@@ -1,3 +1,3 @@
 ops==1.3.0
 etcd3gw==1.0.2
-git+https://github.com/omnivector-solutions/slurm-ops-manager.git@0.8.7
\ No newline at end of file
+git+https://github.com/omnivector-solutions/slurm-ops-manager.git@0.8.9
```

Comment:
Just a clerical question. What changed between slurm_ops_manager 0.8.7 to slurm_ops_manager 0.8.9

Reply:
Hi @NucciTheBoss! It basically drops support for MPI, Singularity and Nvidia installations (https://github.com/omnivector-solutions/slurm-ops-manager/pull/88) and correct default path for plugstack conf (https://github.com/omnivector-solutions/slurm-ops-manager/pull/86)

slurm-ops-manager 0.8.10 applies a small fix (https://github.com/omnivector-solutions/slurm-ops-manager/pull/89)